themes/digital.gov/layouts/partials/core/page-data.html

```diff
@@ -13,7 +13,7 @@
 {{- $email_copy := (print "We value your feedback and open questions. Thank you for helping to make our pages better.") -}}
 
 <div id="page-data" class="branch-{{- .branch -}}" name="page-data">
-  <p class="feedback-copy"><i class="fas fa-question-circle"></i> {{ if .prompt -}}{{- .prompt -}}{{- else -}}Have feedback or questions?{{- end }} <a target="new" href="mailto:{{- if .email -}}{{- .email -}}{{- else -}}digitalgov@gsa.gov{{- end -}}?subject=Feedback%3A%20{{- htmlUnescape .page.Title | safeHTML -}}&amp;body=%0D%0A---enter%20your%20feedback%2Fquestions%20here---%0D%0A%0D%0A%0D%0A-%20-%20-%20-%20-%20%0D%0APage%3A%20{{- htmlUnescape .page.Title | safeHTML -}}%0D%0AURL%3A%20https%3A%2F%2Fwww.digital.gov{{- .page.Permalink | markdownify -}}%0D%0A-%20-%20-%20-%20-%0D%0A%0D%0A%0D%0A{{- $email_copy -}}%0D%0A%0D%0Ahttps%3A%2F%2Fwww.digital.gov%0D%0A%40digital_gov%20on%20Twitter{{- if .cc -}}&amp;cc={{- .cc -}}{{- end -}}">Send&nbsp;us&nbsp;an&nbsp;email&nbsp;»</a></p>
+  <p class="feedback-copy"><i class="fas fa-question-circle"></i> {{ if .prompt -}}{{- .prompt -}}{{- else -}}Have feedback or questions?{{- end }} <a target="new" href="mailto:{{- if .email -}}{{- .email -}}{{- else -}}digitalgov@gsa.gov{{- end -}}?subject=Feedback%3A%20{{- htmlUnescape .page.Title | safeHTML -}}&amp;body=%0D%0A---enter%20your%20feedback%2Fquestions%20here---%0D%0A%0D%0A%0D%0A-%20-%20-%20-%20-%20%0D%0APage%3A%20{{- htmlUnescape .page.Title | safeHTML -}}%0D%0AURL%3A%20https%3A%2F%2Fwww.digital.gov{{- .page.Permalink | markdownify -}}%0D%0A-%20-%20-%20-%20-%0D%0A%0D%0A%0D%0A{{- $email_copy -}}%0D%0A%0D%0Ahttps%3A%2F%2Fwww.digital.gov%0D%0A%40digital_gov%20on%20Twitter{{- if .cc -}}&amp;cc={{- .cc -}}{{- end -}}">Send&nbsp;us&nbsp;an&nbsp;email&nbsp;at&nbsp;digitalgov@gsa.gov</a></p>
```

Comment:
Can we add some formatting to make this more readable?

Also, I don't think we need all of the `&nbsp;` in the link text.